nautobot/extras/tables.py

```diff
@@ -262,6 +268,30 @@ class Meta(BaseTable.Meta):
         )
 
 
+class DynamicGroupTable(BaseTable):
+
+    pk = ToggleColumn()
+    name = tables.LinkColumn(args=[Accessor("slug")])
+    members = tables.Column(accessor="count", verbose_name="Group Members")
+    actions = ButtonsColumn(DynamicGroup, pk_field="slug")
+
+    class Meta(BaseTable.Meta):  # pylint: disable=too-few-public-methods
+        """Resource Manager Meta."""
+
+        model = DynamicGroup
+        fields = (
+            "pk",
+            "name",
+            "description",
+            "content_type",
+            "members",
+            "actions",
+        )
+
+    def render_members(self, value, record):
+        return format_html(f'<a href="{record.get_group_members_url()}">{value}</a>')
```

Comment:
Perhaps, for safety?

```suggestion
        return format_html('<a href="%s">%s</a>', record.get_group_members_url(), value)
```